src/unity/toolkits/util/sframe_test_util.hpp

```diff
@@ -0,0 +1,65 @@
+/* Copyright © 2017 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+ */
+#ifndef TURI_UNITY_SFRAME_TEST_UTIL_H_
+#define TURI_UNITY_SFRAME_TEST_UTIL_H_
+
+#include <boost/test/unit_test.hpp>
+#include <unity/lib/gl_sarray.hpp>
+#include <unity/lib/gl_sframe.hpp>
+#include <util/test_macros.hpp>
+
+#include <string>
+
+__attribute__((__unused__))
+inline bool check_equality_gl_sframe(
```

Comment:
Do we not have this already somewhere? I feel like I've seen this exact code before.